### PR TITLE
buttonRework: Added more customization options to existing button component

### PR DESCRIPTION
**Pull request by carizzuti**
I've added to the existing button component, it will now be able to take `buttonStyle` and `titleStyle` props. This should make it more reusable.

 
image

### CODE REVIEW

File: src/components/button/Button.styles.js [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-        backgroundColor: '#D1D1D1',
+        backgroundColor: color.lightGray
     },
     buttonEnabled: {
-        backgroundColor: '#7199FF',
+        backgroundColor: color.primary
     },
     title: {
+        color: color.white,
         fontSize: 23,
```
Comment (raynesmike) [inline src/components/button/Button.styles.js:18]: nice for changing the color, try to change the sizes too.

**Review by raynesmike — APPROVED** [review]
everything looks good

**Review by Dulaya — APPROVED** [review]
I looked at it last night. We'll put everything together today.

File: src/components/button/Button.styles.js (hunk 1 above)
Comment (carizzuti) [reply src/components/button/Button.styles.js:18]: Would I add a default font size to the size.js theme file?

**Review by aahenry51 — APPROVED** [review]

**Pull request merged**